integration-tests/steps/templates.py

```diff
@@ -96,10 +96,10 @@ def step_impl(context, stack_name):
 
 @then('the output is the same as the contents of "{filename}" template')
```

Comment:
Hmmm, this description should probably be updated to accurate reflect what's being tested